sources.list.jessie

```diff
@@ -0,0 +1,4 @@
+deb http://ftp.debian.org/debian jessie main
+deb http://ftp.debian.org/debian jessie-updates main
```

Comment:
I don't think ftp.debian.org is a good choice...it resolves to a single server in the Netherlands :(.